frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -501,6 +501,8 @@ const ManageHostsPage = ({
   const handleLabelChange = ({ slug }: ILabel): boolean => {
     const { MANAGE_HOSTS } = PATHS;
 
+    const isDeselectingLabel = slug?.slice(7) === selectedLabel?.id.toString();
```

Comment:
What is the significance of the value 7 in this case?

Reply:
The slug string is "labels/{labelId}", so slice it after 7 characters.

Should I add a note?

Reply:
Or maybe replace the `7` with `LABEL_SLUG_PREFIX.length`

Reply:
@gillespi314 ready for re-review!

Reply:
`labelID` now being used which is coming from url params :)